packages/desktop/electron/preload.js

```diff
@@ -55,7 +55,7 @@ try {
             levelFilter: 'debug',
             targetExclusions: ['h2', 'hyper', 'rustls', 'message_handler'],
         }
-        WalletApi.initLogger(loggerOptions)
+        // WalletApi.initLogger(loggerOptions)
```

Comment:
this will remove the console error, but we won't have any logging then, right?
I think its an ok trade-off, because if we need wallet.rs logs, we can then just enable it

Reply:
If there is no explicit reason for them to be commented out, it's best to have them in.

Reply:
I forgot to uncomment it back in :sweat_smile: